packages/ui/src/components/Input/Input.tsx

```diff
@@ -188,13 +188,13 @@ function Input({
 
 export interface TextAreaProps
   extends Omit<React.InputHTMLAttributes<HTMLTextAreaElement>, 'size' | 'onCopy'> {
-  descriptionText?: string
+  descriptionText?: string | React.ReactNode | undefined
```

Comment:
Brought these types in line with `InputProps`, because they all get passed straight to `FormLayout` anyway so they're handled the same.